modules/aruco/tutorials/aruco_board_detection/aruco_board_detection.markdown

```diff
@@ -198,7 +198,9 @@ Finally, a full example of board detection:
 
 Sample video:
 
-[![Board Detection video](http://img.youtube.com/vi/Q1HlJEjW_j0/0.jpg)](https://youtu.be/Q1HlJEjW_j0)
+@htmlonly
+<iframe width="420" height="315" src="https://www.youtube.com/embed/Q1HlJEjW_j0" frameborder="0" allowfullscreen></iframe>
+@endhtmlonly
```

Comment:
Does this work?
```
[[embed url=https://www.youtube.com/embed/Q1HlJEjW_j0]]
```

Described here: https://sourceforge.net/p/doxygen/discussion/markdown_syntax#md_ex_video